src/calculator.js

```diff
@@ -1,47 +1,31 @@
-exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
+/* eslint linebreak-style: ["error", "windows"] */
+exports._check = (x, y) => {
+  // Function created to check that inputs are numbers
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
+};
+
+exports.add = (x, y) => {
+  exports._check(x, y);
   return x + y;
 };
 
 exports.subtract = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  exports._check(x, y);
   return x - y;
 };
```

Comment:
I saw "line breaks" issue because I am on a Windows machine when I ran the "rpm run lint" command .  That is why I added the /* eslint linebreak-style: ["error", "windows"] */ to both the "calculator.js" and "calculator.test.js" files.  If I removed that line in both the files the numerous "Expected linebreaks to 'CRLF' but found 'LF'." errors on your Mac or UNIX machine will disappear, but will remain on my machine :-)  Your suggestion please.